src/modules/Dashboard/features/Applet/Overview/QuickStats/QuickStats.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box } from '@mui/material';
+
+import { Svg } from 'shared/components';
+import { StyledFlexColumn, StyledHeadlineLarge, variables } from 'shared/styles';
+
+import { StyledRoot } from './QuickStats.styles';
+import { QuickStatsProps } from './QuickStats.types';
+
+export const QuickStats = ({ stats = [], ...otherProps }: QuickStatsProps) => (
+  <StyledRoot component="ul" {...otherProps}>
+    {stats.map(({ label, icon, value, sx, ...otherStatProps }, i) => (
+      <StyledFlexColumn
+        component="li"
+        key={i}
+        sx={{ gap: 1.2, flexGrow: 1, position: 'relative', ...sx }}
+        {...otherStatProps}
+      >
+        <Svg height="24" fill={variables.palette.outline_variant} id={icon} width="24" />
```

Comment:
This should be `on_surface_variant` according to [Figma](https://www.figma.com/design/6BNx63peNVzrHAF1FoZ78x/Multi-Informant-2024?node-id=1761-40649&t=pntZXC2cWLgB76WL-4):
```suggestion
        <Svg height="24" fill={variables.palette.on_surface_variant} id={icon} width="24" />
```